tests/misc_env/install_prereq.py

```diff
@@ -423,6 +423,10 @@ def registry_login(ceph, distro_ver):
     auths_dict = {"auths": auths}
     ceph.exec_command(sudo=True, cmd="mkdir -p ~/.docker")
     ceph.exec_command(cmd="mkdir -p ~/.docker")
+    ceph.exec_command(cmd="sudo grep -q '^\[\[registry\]\]' /etc/containers/registries.conf && \
+                      sudo sed -i /etc/containers/registries.conf || echo -e '\n[[registry]]\nlocation = \"registry-....\"\ninsecure = true' | \
```

Comment:
Change `registry-....` with `registry-proxy.engineering.redhat.com`

Reply:
Hey @vamahaja , @obannak 
Can we please wait on this commit, as we are discussing few things on multi registry access,
-  Removal of docker directory is part of it.
- working on different mechanism

cc: @psathyan

Reply:
Sure @sunilkumarn417